packages/cadl-ranch-specs/cadl-ranch-summary.md

```diff
@@ -1578,6 +1578,16 @@ Scenario that returns a different file encoding depending on the accept header.
 - image/png return a png image
 - image/jpeg return a jpeg image
 
+### Payload_MediaType_StringBody
+
+- Endpoints:
+  - `post /payload/media-type/stringBody/sendAsText`
+  - `post /payload/media-type/stringBody/getAsText`
+  - `post /payload/media-type/stringBody/sendAsJson`
+  - `post /payload/media-type/stringBody/getAsJson`
+
+When the request body and response body are both strings, the different media types would lead to different payloads.
```

Comment:
Personally prefer to use "content type" in all places.

Reply:
I use "media type" because we not only contain "content type" but also "accept".